src/screens/UserManagement/UserRoleEntry.res

```diff
@@ -54,6 +74,11 @@ let make = () => {
     if usersData->Array.length === 0 {
```

Comment:
there is no point in adding this check always usersData is going to be empty

Reply:
Theres a API call for USERS_LIST in ShowUsersData. So if somebody come from there (via clicking back) by that time usersData would already be present.